src/NUnitEngine/Addins/nunit-v2-result-writer/NUnit2ResultSummary.cs

```diff
@@ -101,7 +100,6 @@ private void Summarize(XmlNode node)
                         case "Ignored":
                             ignoreCount++;
                             break;
-                        case "Skipped":
```

Comment:
It is Redundant code.

On Saturday, April 11, 2015, CharliePoole <notifications@github.com> wrote:

> In src/NUnitEngine/Addins/nunit-v2-result-writer/NUnit2ResultSummary.cs
> <https://github.com/nunit/nunit/pull/586#discussion_r28157984>:
>
> > @@ -101,7 +100,6 @@ private void Summarize(XmlNode node)
> >                          case "Ignored":
> >                              ignoreCount++;
> >                              break;
> > -                        case "Skipped":
>
> Not needed except for understanding on the part of the reader. :-)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/nunit/nunit/pull/586/files#r28157984>.
>